es6.md

```diff
@@ -3,70 +3,71 @@ layout: default
-features from the [JavaScript ECMA-262
-specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm)
-are brought to io.js developers in a timely manner, as well as continued
-performance and stability improvements.
-
-Version {{ iojs_version }} of io.js ships with V8 {{ v8_version }}, which
-includes ES6 features well beyond version 3.26.33 that will be shipped with
-joyent/node@0.12.x.
-
-## No more --harmony flag
-
-On joyent/node@0.12.x (V8 3.26), the `--harmony` runtime flag enabled all
-**completed**, **staged** and **in progress** ES6 features together, in bulk
-(with the exception of nonstandard/non-harmonious semantics for `typeof` which
-were hidden under `--harmony-typeof`). This meant that some really buggy or even
-broken features like
-[proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
-were just as readily available for developers as
+# ES6 i io.js
+
+io.js bruker moderne versjoner av [V8](https://code.google.com/p/v8/).
+Ved å holde prosjektet oppdatert med de siste versjonene av V8 sørger
+vi for at nye funksjoner fra [JavaScript 
+ECMA-262 spesifikasjonen](http://www.ecma-international.org/publications/standards/Ecma-262.htm)
+er tilgjengelig for io.js utviklere innen rimelig tid. I tillegg vil sikkerhets-
+og ytelesesforbedringer komme raskt.
+
+Versjon {{ iojs_version }} av io.js kommer med V8 {{ v8_version }}, denne
+inkluderer ES6-funksjoner godt forbi versjon 3.28.73 som vil bli levert med
+Node.js™ 0.12.x.
+
+## Slutt på --harmony-flagget
+
+Med Node.js™@0.12.x (V8 3.28+) aktiverte  `--harmony` flagget alle 
+**completed**, **staged** og **in progress** ES6-funksjoner sammen
+(med unntak av `proxies` som 
+ble skjult ved bruk av `--harmony-proxies`). Dette resulterte i at ustabile eller 
+til og med ødelagte funksjoner som 
+[Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+var like tilgjengelig for utviklere som 
 [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*),
-which had very little or even no known-issues. As such, it was best practice to
-either enable only certain features by using specific runtime harmony feature
-flags (e.g. `--harmony-generators`), or simply enable all of them and then use a
-restricted subset.
+som hadde veldig få eller ingen kjente problemer. Følgelig var det sikrest
+å aktivere enten et fåtall funksjoner ved å bruke spesifikke harmony-flagg (f.eks.
+`--harmony-generators`), eller aktivere dem alle og deretter bare bruke
+en begrenset delmengde.
 
-With io.js@1.x (V8 4.1+), all that complexity goes away. All harmony features
-are now logically split into three groups for **shipping**, **staged** and **in
-progress** features:
+Med io.js@1.x (V8 4.1+) forsvinner all denne kompleksiteten. Alle harmony-funksjoner
+er nå logisk fordelt inn i tre grupper for **shipping**, **staged** og **in
```

Comment:
> Forøvrig burde det kanskje stå bindestrek etter fremmedord for å unngå orddelingsfeil, som f.eks.
io.js versjon -> io.js-versjon
--es-staging flagget -> --es-staging-flagget
in-progress funksjoner -> in-progress-funksjoner

I såfall burde nok "shipping, staged og in-progress funksjoner" skrives om til "shipping-, staged- og in-progress-funksjoner".